chef_master/source/dk_windows.rst

```diff
@@ -1,101 +1,195 @@
-
-.. note:: `Chef Workstation <https://downloads.chef.io/chef-workstation/>`__ gives you everything you need to get started with Chef — ad hoc remote execution, remote scanning, configuration tasks, cookbook creation tools as well as robust dependency and testing software — all in one easy-to-install package. Chef Workstation replaces ChefDK, combining all the existing features with new features, such as ad-hoc task support and the new Chef Workstation desktop application. Chef will continue to maintain ChefDK, but new development will take place in Chef Workstation without backporting features.
-
-Workstations
+Chef Workstation
 =====================================================
-.. tag workstation_summary
 
-A workstation is a computer running Chef Workstation that is used to author cookbooks, interact with the Chef server, and interact with nodes.
+Start your infrastructure automation quickly and easily with `Chef Workstation <https://www.chef.sh/>`__ Chef Workstation gives you everything you need to get started with Chef — ad hoc remote execution, remote scanning, configuration tasks, cookbook creation tools as well as robust dependency and testing software — all in one easy-to-install package.
 
-The workstation is where users do most of their work, including:
+Chef Workstation replaces ChefDK, combining all the existing features with new features, such as ad-hoc task support and the new Chef Workstation desktop application. Chef will continue to maintain ChefDK, but new development will take place in Chef Workstation without back-porting features.
 
-* Developing and testing cookbooks and recipes
-* Testing Chef code
-* Keeping the Chef repository synchronized with version source control
-* Configuring organizational policy, including defining roles and environments, and ensuring that critical data is stored in data bags
-* Interacting with nodes, as (or when) required, such as performing a bootstrap operation
+`Download Chef Workstation <https://https://downloads.chef.io/chef-workstation/>`__ 
 
-.. end_tag
-
-Installing ChefDK on Windows
+Chef Workstation Supported Platforms
+-----------------------------------------------------
 
-Configure Environment
+Supported Host Operating Systems:
+
+.. list-table:: Supported Host Operating System
+   :widths: 15 15 
+   :header-rows: 1
+  
+   * - Platform
+     - Version
+   * - Apple macOS
+     - 10.11, 10.12, 10.13
+   * - Microsoft Windows
+     - 10, Server 2008 R2, Server 2012, Server 2012 R2, Server 2016
+   * - Red Hat Enterprise Linux / CentOS
+     - 6.x, 7.x
+   * - SUSE Enterprise Linux Server
+     - 11 SP4, 12 SP1+
+   * - Ubuntu 
+     - 14.04, 16.04, 18.04
+   * - Debian
+     - 7.x, 8.x, 9.x
+
+.. list-table:: Supported Target Operating System
+   :widths: 15 15 
+   :header-rows: 1
+
+   * - Platform
+     - Version
+   * - Microsoft Windows
+     - 10, Server 2008 R2, Server 2012, Server 2012 R2, Server 2016
```

Comment:
This needs re-syncing with the current platforms (e.g. to include Windows Server 2019)

Reply:
done